pkg/policy/api/l4.go

```diff
@@ -199,3 +229,58 @@ func (rules *L7Rules) Len() int {
 func (rules *L7Rules) IsEmpty() bool {
 	return rules == nil || (rules.HTTP == nil && rules.Kafka == nil && rules.DNS == nil && rules.L7 == nil)
 }
+
+// PortRules is a slice of PortRule
```

Comment:
```suggestion
// PortRules is a slice of PortRule.
```

These fields potentially may show in the CRD, and this description will be embedded there. We should be consistent with our formatting.